exonum/src/messages/spec.rs

```diff
@@ -12,89 +12,75 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/// `message!` specifies a datatype for digitally signed messages that can be sent
```

Comment:
Do I understand correctly that it would be relatively easy to leave `message!` as is? If so, I'd recommend to do that in the light of the instability of `transactions!` macro that we've discussed previously.